### DISCUSSION TITLE
Apply Palette feature

### DISCUSSION THREAD

**Discussion by londiebrondie** [Orama-Interactive/Pixelorama#824] [Ideas]
I was wondering if you guys could add an "apply palette" feature. I noticed that https://www.pixilart.com/draw has this feature. And no, I'm not talking about gradients.

**Comment by OverloadedOrama** [comment]
From what I understand, the apply palette feature maps each color of the image to the nearest color of the selected palette. I added a Palettize image effect a few days ago for version 1.0 in https://github.com/Orama-Interactive/Pixelorama/commit/1c9c8bf4e3c38e1e9dfae521d256a35754a610c1, which does this.
